strategies.py

```diff
@@ -43,14 +43,42 @@ def rebel(mymoves,opmoves): #uses the most popular move
     if(hawk_count > len(opmoves) / 2): return "dove"
     else: return "hawk"
 
-def pushover(mymoves,opmoves): #always hawk unless last 3 opponents were hawks
+def pushover(mymoves,opmoves): #always dove unless last 3 opponents moves were hawks | Lukas
     for i in range(min(3,len(opmoves))):
         if(opmoves[len(opmoves)-1-i] == "dove"): return "dove"
 
     return "hawk"
 
-def guardian(mymoves,opmoves): #always hawk unless last 3 opponents were hawks
+def guardian(mymoves,opmoves): #always hawk unless last 3 opponents were dove | Lukas
     for i in range(min(3,len(opmoves))):
         if(opmoves[len(opmoves)-1-i] == "hawk"): return "hawk"
 
-    return "dove"     
\ No newline at end of file
+    return "dove"     
+
+def believer(my, op): #tiki praeitimi, renkasi ta move kuris anksciau geras jam buvo | Tomas
+    matrix = [[(5,5),(2,7)],[(7,2),(0,0)]]
+    d={'dove':0,'hawk':1}
+    if len(my)<10:
+        return 'dove' if random.random()<0.5 else 'hawk'
+    gains=[0,0]
+    for i in zip(my,op):
+        if d[i[0]]:
+            gains[1]+=matrix[d[i[0]]][d[i[1]]][0]
+        else:
+            gains[0]+=matrix[d[i[0]]][d[i[1]]][0]
+    if gains[1]>gains[0]:
+        return 'hawk'
+    else:
+        return 'dove'
+
+def alzheimer_conformist(mymoves,opmoves): #uses the most popular move in the last five moves | Mokslo gildija
+    if(len(mymoves)==0): return "dove"
```

Comment:
this isn't c, you don't need parens around conditions, loops, etc ;)
PEP8 my beloved xD